### PR TITLE
As an SDK user, I want to be able get app information from the SDKs

### PR DESCRIPTION
**Pull request by calhorn95**

### CODE REVIEW

File: passageidentity/passage.py [hunk 1]
```diff
@@ -183,6 +197,14 @@ def createMagicLink(self, magicLinkAttributes: MagicLinkAttributes) -> Union[Pas
         except Exception as e:
             raise PassageError("Could not create magic link")
 
+    """
+    Use Passage API to get info for their app.
+    """
+
+    def getApp(self) -> Union[PassageAppType, PassageError]:
+        app_info = fetchApp(self.app_id)
```
Comment (dylanbrookes) [inline passageidentity/passage.py:205]: Has this been tested to see what is returned when an error is thrown by`fetchApp()`? Does the error go into the app_info var or does it properly bubble up the stack to throw the error?

File: tests/authenticate_test.py [hunk 2]
```diff
@@ -54,6 +54,12 @@ def testFlaskInValidToken():
         with pytest.raises(PassageError) as e:
             user = psg.authenticateRequest(request)
 
+def testGetApp():
+    psg = Passage(PASSAGE_APP_ID, PASSAGE_API_KEY)
+    app = psg.getApp()
+    assert PASSAGE_APP_ID == app.id
```
Comment (dylanbrookes) [inline tests/authenticate_test.py:60]: Thoughts on adding a test to check failure case to make sure the error is propagating properly?

**Review by dylanbrookes — APPROVED** [review]
Looks good; some small comments

File: passageidentity/passage.py (hunk 1 above)
Comment (calhorn95) [reply passageidentity/passage.py:205]: it throws the error properly

**Pull request merged**